postject-api.h

```diff
@@ -33,6 +34,11 @@ inline void postject_options_init(struct postject_options* options) {
   options->pe_resource_name = NULL;
 }
 
+static inline bool postject_has_resource() {
+  const char* sentinel = "POSTJECT_SENTINEL_fce680ab2cc467b6e072b8b5df1996b2:0";
+  return sentinel[strlen(sentinel) - 1] == '1';
```

Comment:
Is that won't always return `false`?

Reply:
nope because if you look at the change in `src/api.js`, we are changing the last character in the sentinel string from `0` to `1`